package.json

```diff
@@ -8,7 +8,7 @@
     "format:base": "prettier --config ./.prettierrc \"*.{json,md}\" \"src/**/*.{css,scss,ts}\" \"test/**/*.{css,scss,ts}\"",
     "format:check": "npm run format:base -- --list-different",
     "format:fix": "npm run format:base -- --write",
-    "lint": "tslint -c tslint.json \"src/**/*.ts\" \"test/**/*.ts\"",
+    "lint": "tslint -p . -c tslint.json \"src/**/*.ts\" \"test/**/*.ts\"",
```

Comment:
Do we need `-p`? Are we using any rules with type checking?

Reply:
I see that's required because of the `deprecation` rule?

Reply:
Yes, `deprecation` rule requires type information.